tools/src/h5dump/h5dump_xml.c

```diff
@@ -4529,12 +4526,11 @@ xml_print_enum(hid_t type)
                 h5tools_str_append(&buffer, "%02x", value[i * dst_size + j]);
         }
         else if (H5T_SGN_NONE == H5Tget_sign(native)) {
-            h5tools_str_append(&buffer, "%" H5_PRINTF_LL_WIDTH "u",
-                               *((unsigned long long *)((void *)(value + i * dst_size))));
+            /* XXX haven't I fixed this code somewhere else, before? */
```

Comment:
XXX?

Reply:
@gnuoyd is this comment to be kept, or was it a temporary note?